axelrod/player.py

```diff
@@ -3,9 +3,10 @@
 import random
 import copy
 
-from axelrod.actions import Actions, flip_action
+from axelrod.actions import Actions, flip_action, Action
 from .game import DefaultGame
 
+from typing import Dict, Any
```

Comment:
pedantic: alphabetize imports, move typing import above to the section with the standard library imports (this is in PEP8 IIRC)